### PR TITLE
Improve error handling of ttyp object handler

### PR DESCRIPTION
**Pull request by absap**
I came across the TTYP object handler and did some improvements of the error messages that could be "thrown" by the function modules. This approach with a Message class could be extended to others object handlers and parts of the code.

Relates to issue #1972

### CODE REVIEW

**Comment by absap** [comment]
Indeed, didn't think about it.
Also started working on an alternative.

**Comment by absap** [comment]
I'm not sure if string templates are supported in 702? If yes, I would refactor the coding again.
```abap
lv_msg = |Error ... on object { lv_name } ...|
```

**Comment by larshp** [comment]
string templates work in 702, https://syntax.abaplint.org/?filter=string#/expression/StringTemplate

File: src/zabapgit_objects.msag.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_MSAG" serializer_version="v1.0.0">
```
Comment (larshp) [inline src/zabapgit_objects.msag.xml]: plus delete this file

**Pull request merged**